sql/rename.go

```diff
@@ -136,3 +136,57 @@ func (p *planner) RenameTable(n *parser.RenameTable) (planNode, error) {
 
 	return &valuesNode{}, nil
 }
+
+// RenameIndex renames the index.
+// Privileges: WRITE on database.
```

Comment:
yup. we need WRITE on the table descriptor.
@wenyong-h can you also add the following to this comment:
```
//   notes: postgres requires CREATE on the table.
//              mysql requires ALTER, CREATE, INSERT on the table.